src/panels/config/scene/ha-config-scene.ts

```diff
@@ -38,9 +38,12 @@ class HaConfigScene extends HassRouterPage {
 
   private _computeScenes = memoizeOne((states: HassEntities) => {
     const scenes: SceneEntity[] = [];
-    Object.keys(states).forEach((entityId) => {
-      if (computeDomain(entityId) === "scene") {
-        scenes.push(states[entityId] as SceneEntity);
+    Object.values(states).forEach((state) => {
+      if (
+        computeDomain(state.entity_id) === "scene" &&
```

Comment:
Just so you know, we also have a `computeStateDomain` I remember now. It does just this.